chef/data_bags/crowbar/migrate/network/101_add_ironic_network.rb

```diff
@@ -0,0 +1,20 @@
+def upgrade(ta, td, a, d)
+  unless a["networks"].key? "ironic"
+    a["networks"]["ironic"] = ta["networks"]["ironic"]
+  end
+
+  # copy intf1 as intf3
+  a["conduit_map"].each do |cm|
+    unless cm["conduit_list"].key? "intf3"
+      cm["conduit_list"]["intf3"] = cm["conduit_list"]["intf1"]
```

Comment:
> This is exactly why I introduced intf3 instead of using the existing conduits.

Yeah, I am just wondering about the schema migration, because there you assign the ```intf3``` interface to exactly the same physical interfaces as ```intf1```. I am not sure if there is a good solution to this, other than not touching the conduit_maps at all in the migration and document it. As you already mentioned, in practice this will most likely need a custom config anyway.